### PR TITLE
Enable testing of switching off VFS for all VFSses

### PR DESCRIPTION
**Pull request by erikjv**
The test `TestSyncVirtualFiles::testWipeVirtualSuffixFiles` was only run with suffix VFS, but is also applicable for other VFSses. This change renames it to `testSwitchOffVFS` and runs it on all available VFS plugins.

Fixes: https://github.com/owncloud/client/issues/11342

### CODE REVIEW

**Comment by erikjv** [comment]
Note: this does not pass the tests when winvfs is available!

File: test/testsyncvirtualfiles.cpp [hunk 1]
```diff
@@ -882,27 +906,34 @@ private slots:
...
+        QVERIFY(fakeFolder.currentLocalState().find(fileName(QStringLiteral("f1"))));
+        QVERIFY(fakeFolder.currentLocalState().find(fileName(QStringLiteral("A/a1"))));
+        QVERIFY(fakeFolder.currentLocalState().find(fileName(QStringLiteral("A/a3"))));
+        QVERIFY(fakeFolder.currentLocalState().find(fileName(QStringLiteral("A/B/b1"))));
 
         // Make local changes to a3
-        fakeFolder.localModifier().remove(QStringLiteral("A/a3") + Theme::instance()->appDotVirtualFileSuffix());
-        fakeFolder.localModifier().insert(QStringLiteral("A/a3") + Theme::instance()->appDotVirtualFileSuffix(), 100_b);
+        fakeFolder.localModifier().remove(fileName(QStringLiteral("A/a3")));
+        fakeFolder.localModifier().insert(fileName(QStringLiteral("A/a3")), 100_b); // TODO: should this really have the suffix-vfs file suffix??
```
Comment (erikjv) [inline test/testsyncvirtualfiles.cpp:916]: @TheOneRing What do you think?

**Comment by TheOneRing** [comment]
ci looks good: https://github.com/owncloud/client-desktop-vfs-win/actions/runs/7020419509/job/19100078148

File: test/testsyncvirtualfiles.cpp (hunk 1 above)
Comment (TheOneRing) [reply test/testsyncvirtualfiles.cpp:916]: Can you check what happened here previously?
With suffix files that should result in an invalid placeholder that was no longer synced.
So after this change the local and remote state should differ, we should have a "A/a3.owncloud" locally but no corresponding file on the remote.

**Comment by erikjv** [comment]
The test also passes with WinVFS: https://github.com/owncloud/client-desktop-vfs-win/actions/runs/7118184195

**Review by TheOneRing — APPROVED** [review]

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**